site/app/templates/course/EditCourseMaterialsForm.twig

```diff
@@ -0,0 +1,116 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-course-materials-form{% endblock %}
+{% block title %}Edit Course Materials{% endblock %}
+{% block body %}
+    <div id="material-edit-form" class="flex-col flex-col-space" data-directory = "Check">
+        <label id="edit_sections">
+            <p class="black-message">
+                Do you want to restrict this course material to some sections?
+            </p>
+            <label class = "radio">
+                <input id = "all_Sections_Showing_No" name="show_Some_Section_Selection_Edit" type = "radio" onchange="toggleRegistrationSectionsEdit()" checked>
```

Comment:
ids should be in `kebab-case`. Both name and id should be all in lowercase. Carry down through all of this file.